scripts/simpleresponses.coffee

```diff
@@ -0,0 +1,54 @@
+# Description:
+#   Ein paar weitere kleine Antworten von matthias
+#
+# Dependencies:
+#   None
+#
+# Configuration:
+#   None
+#
+# Commands:
+#   hubot bash me - Frag' hubot nach einem random Zitat link
+#   hubot pizza - hubot nennt alle Details für eine Pizzabestellung (im Ratsaal)
+#   hubot filmlist - Frag' hubot nach Links zu beiden Filmlisten
+#
+#
+# Author:
+#   kiliankoe
+
+
+simpleresponses = [
+  [
+    /pizza/i,
+    "Ich bin Matthias Stuhlbein,
+    Nöthnitzerstr. 46, 01187 Dresden.
+    Fakultät Informatik. Mail: pizza@ifsr.de,
+    Telefon: 0351 46338223 - Pizzen schneiden nicht vergessen ;)"
```

Comment:
Using actual newlines makes the output look like this:
```
Ich bin Matthias Stuhlbein,    Nöthnitzerstr. 46, 01187 Dresden.[...]
```
instead of
```
Ich bin Matthias Stuhlbein,
Nöthnitzerstr. 46, 01187 Dresden.[...]
```

`/n` is the preferred way to go.